File: scratch/count_tags.py
with open('c:\\Users\\LOQ\\Desktop\\hirebridge-employer-review-page-fixed\\src\\app\\features\\employer\\review-candidates\\review-candidates-page.component.html', 'r', encoding='utf-8') as f:
    content = f.read()
    open_tags = content.count('<div')
    close_tags = content.count('</div')
    print(f"Open: {open_tags}, Close: {close_tags}")

File: scratch/count_tags.js
const fs = require('fs');
const content = fs.readFileSync('c:\\Users\\LOQ\\Desktop\\hirebridge-employer-review-page-fixed\\src\\app\\features\\employer\\review-candidates\\review-candidates-page.component.html', 'utf8');
const openTags = (content.match(/<div/g) || []).length;
const closeTags = (content.match(/<\/div/g) || []).length;
console.log(`Open: ${openTags}, Close: ${closeTags}`);
